packages/wallet/src/redux/internal/actions.ts

```diff
@@ -22,4 +22,23 @@ export const directFundingConfirmed = (channelId: string) => ({
 });
 export type DirectFundingConfirmed = ReturnType<typeof directFundingConfirmed>;
 
-export type InternalAction = DirectFundingRequested | DirectFundingConfirmed;
+export const OPEN_CHANNEL_SUCCESS = 'WALLET.INTERNAL.OPEN_CHANNEL_SUCCESS';
+export const openChannelSuccess = (channelId: string) => ({
+  type: OPEN_CHANNEL_SUCCESS as typeof OPEN_CHANNEL_SUCCESS,
+  channelId,
+});
+export type OpenChannelSuccess = ReturnType<typeof openChannelSuccess>;
+
+// TODO: is the last committment needed?
```

Comment:
`channelId` is ambiguous here: is it the id of the ledger channel, or the channel allocated to once consensus was reached?

This action might need both. But I think it might be pragmatic to include the entire allocation, before and after consensus was reached. That way, an application channel can filter `CONSENSUS_REACHED` events based on whether its funding was changed.